packages/beapi/src/command/executeCommand.ts

```diff
@@ -1,31 +1,69 @@
-import { world } from 'mojang-minecraft'
-import type { ExecuteCommandResponse, Dimensions } from '../@types/BeAPI.i'
+import { world } from "mojang-minecraft";
```

Comment:
We utilize single quoting conventions. Please ensure eslint and prettier are enabled.